### ISSUE TITLE
SlevomatCodingStandard.Namespaces.UnusedUses mistakenly reports E_ALL

### ISSUE THREAD

**Issue by morozov** [slevomat/coding-standard#1038]
```
$ composer show | grep -P 'coding-standard|php_codesniffer'

doctrine/coding-standard                       8.0.0   The Doctrine Coding Standard is a ...
slevomat/coding-standard                       6.3.8   Slevomat Coding Standard for PHP_C...
squizlabs/php_codesniffer                      3.5.5   PHP_CodeSniffer tokenizes PHP, Jav...
```
Consider the following code example:
```php
<?php

namespace Foo;

use const E_ALL;
use const E_NOTICE;

echo E_ALL & ~E_NOTICE;
```
The following violation is reported:
```
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 5 | ERROR | [x] Type E_ALL is not used in this file.
   |       |     (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------
```
After a fix, the file will look like this:
```php
<?php

namespace Foo;

use const E_NOTICE;

echo E_ALL & ~E_NOTICE;
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/c4e287879af90a2f56b1b516deaba707858c9a24
